scripts/create-feature-annotations.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
```

Comment:
There's not enough documentation here yet for me to quickly understand the purpose of each script and how they're intended to be used.